.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,52 @@
+{
+    "configurations": [
+        {
+            "name": "Linux",
+            "includePath": [
+                "${workspaceFolder}/**",
+                "/usr/include/**",
+                "/usr/local/include/**"
+            ],
+            "defines": [],
+            "compilerPath": "/usr/bin/gcc",
+            "cStandard": "c11",
+            "cppStandard": "c++11",
+            "intelliSenseMode": "gcc-x64",
+            "compileCommands": "${workspaceFolder}/build.debug/compile_commands.json",
+            "browse": {
+                "path": [
+                    "${workspaceFolder}/build.debug/",
+                    "${workspaceFolder}/mscore/",
+                    "${workspaceFolder}/libmscore/",
+                    "${workspaceFolder}/aeolus/",
+                    "${workspaceFolder}/assets/",
+                    "${workspaceFolder}/audiofile/",
+                    "${workspaceFolder}/awl/",
+                    "${workspaceFolder}/build/",
+                    "${workspaceFolder}/bww2mxml/",
+                    "${workspaceFolder}/crashreporter/",
+                    "${workspaceFolder}/demos/",
+                    "${workspaceFolder}/doc/",
+                    "${workspaceFolder}/effects/",
+                    "${workspaceFolder}/fluid/",
+                    "${workspaceFolder}/fonts/",
+                    "${workspaceFolder}/fonttools/",
+                    "${workspaceFolder}/manual/",
+                    "${workspaceFolder}/midi/",
+                    "${workspaceFolder}/miditools/",
+                    "${workspaceFolder}/mscore/",
+                    "${workspaceFolder}/mtest/",
+                    "${workspaceFolder}/rdoc/",
+                    "${workspaceFolder}/share/",
+                    "${workspaceFolder}/synthesizer/",
+                    "${workspaceFolder}/test/",
+                    "${workspaceFolder}/thirdparty/",
+                    "${workspaceFolder}/vtest/",
+                    "${workspaceFolder}/zerberus/",
+                    "${workspaceFolder}/*"
```

Comment:
BTW, the reason I've specified almost every directory here, is because I want to omit `/omr`. Including this directory breaks IntelliSense, because it can't find it in compile commands since we compile without it. Unfortunately there's no 'ignore directory' settings - you can only explicitly tell it which directories to browse.